### PR TITLE
phpspec: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/phpspec.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# phpspec
+
+> A SpecBDD testing framework for PHP.
+> More information: <https://phpspec.net>.
+
+- Describe a new class specification:
+
+`phpspec describe {{class_name}}`
+
+- Run all specficiations in the "spec" directory:
```
Comment (sbrl) [inline pages/common/phpspec.md]: Typo?

File: pages/common/phpspec.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`phpspec run`
+
+- Run a specific specification:
```
Comment (sbrl) [inline pages/common/phpspec.md]: What a tongue twister :P

File: pages/common/phpspec.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/phpspec.md]: :facepalm: Yep, damn it.

File: pages/common/phpspec.md (hunk 2 above)
Comment (andrik) [reply pages/common/phpspec.md]: Maybe this one sounds better 

> Run a single specification

File: pages/common/phpspec.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# phpspec
+
+> A SpecBDD testing framework for PHP.
+> More information: <https://phpspec.net>.
+
+- Describe a new class specification:
```
Comment (andrik) [inline pages/common/phpspec.md]:
```suggestion
- Creates a specification for a class:
```

File: pages/common/phpspec.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# phpspec
+
+> A SpecBDD testing framework for PHP.
```
Comment (andrik) [inline pages/common/phpspec.md]:
```suggestion
> A Behaviour Driven Development tool for PHP.
```

File: pages/common/phpspec.md (hunk 2 above)
Comment (pxgamer) [reply pages/common/phpspec.md]:
```suggestion
- Run a single specification:
```

**Pull request merged**